tools/eslint-rules/rules/use-default-provide-config.ts

```diff
@@ -28,7 +28,7 @@ export const rule = ESLintUtils.RuleCreator(() => __filename)({
     type: 'problem',
     docs: {
       description: ``,
-      recommended: 'error',
+      recommended: 'recommended',
```

Comment:
Why such a change?
Does it change the severity to lower, e.g. from error to warning/info?

Reply:
Such a change was forced by changes in the API of `@typescript-eslint`: https://typescript-eslint.io/blog/announcing-typescript-eslint-v6/#reworked-configuration-names

The `recommended` value is what we need now. It doesn't affect the severity as we set the level of this rule in `.eslintrc.json` file: https://github.com/SAP/spartacus/blob/57f492f8802b918ea11c48414d87927e9373fe6f/.eslintrc.json#L111